tests/e2e/scripts/helm_keadm_e2e.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+KUBEEDGE_ROOT=$PWD
+WORKDIR=$(dirname $0)
+HELM_DIR=$KUBEEDGE_ROOT/build/helm
+E2E_DIR=$(realpath $(dirname $0)/..)
+IMAGE_TAG=$(date +%Y%m%d)
```

Comment:
this is not precise, if we trigger the CI twice in one day, Isn't this conflict with each other, IMHO, we'd better use some others like git commit info.